data-source/json/test_schema_theme_census.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
There are already some schemas that test themes named `test_theme_*`, this should be named `test_theme_census` to be consistent.